### PR TITLE
Adds webpack and build support

### PR DESCRIPTION
**Pull request by bipuladh**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,32 +3,49 @@
...
+    "webpack-dev-server": "^4.15.0"
   },
   "consolePlugin": {
     "name": "fusion-console",
     "version": "0.0.0",
     "displayName": "Fusion Console Plugin",
     "description": "Console Plugin for ODF + Fusion Integrated UI.",
-    "exposedModules": {},
+    "exposedModules": {
+      "features": "@fusion/core/Features"
```
Comment (SanjalKatiyar) [inline package.json]: file name
```suggestion
      "features": "@fusion/core/features"
```

File: package.json [hunk 2]
```diff
@@ -3,32 +3,49 @@
   "version": "0.0.0",
   "private": true,
   "license": "Apache-2.0",
-  "scripts": {},
+  "scripts": {
+    "build": "yarn ts-node node_modules/.bin/webpack -c ./webpack.config.ts",
```
Comment (SanjalKatiyar) [inline package.json:7]: no bifurcation on "production" or "development" NODE_ENV ??

File: package.json (hunk 1 above)
Comment (bipuladh) [reply package.json]: lets decide this for later. I would rather have filenames start with Upper case and folders with lowercase.

File: package.json (hunk 2 above)
Comment (bipuladh) [reply package.json:7]: `NODE_ENV` can be set as env variable in dockerfile and it will pick from there.

File: package.json (hunk 1 above)
Comment (ivoalmeida) [reply package.json]: According to what we discussed, I think this be features

File: packages/fusion/Features.ts
Comment (ivoalmeida) [inline packages/fusion/Features.ts:1]: According to what we discussed, I think this be features

**Pull request merged**